queue_job/models/queue_job.py

```diff
@@ -506,3 +508,25 @@ def _test_job(self, failure_rate=0):
         _logger.info("Running test job.")
         if random.random() <= failure_rate:
             raise JobError("Job failed")
+
+    def _call_api(self, **kwargs):
+        webhook_url = (
+            self.env["ir.config_parameter"].sudo().get_param("queue_job.webhook_url")
```

Comment:
why? better if it can be configurable as well as part of kwargs as one might need a different webhook_url per function